modules/payments/backend/modules/payments/README.md

```diff
@@ -0,0 +1,4 @@
+
+# Dependencies
+
+-  Stripe
```

Comment:
Please document what's required before running the project. I'd imagine `python manage.py migrate` at the very least

Reply:
And regarding Stripe, can you explain what's needed or link the docs?